src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java

```diff
@@ -24,12 +24,14 @@
      * Type B is a class that extends {@link AbstractBuildingWorker.View}.
      */
     protected final B          building;
```

Comment:
### Errors:
- Protected field building 
- Can be package-private

### Warnings:
- Instance field name building doesn't match regex 'm_[a-z][A-Za-z\d]*'